connector_search_engine/models/se_binding.py

```diff
@@ -12,9 +14,17 @@
 
 class SeBinding(models.AbstractModel):
     _name = "se.binding"
-    _se_model = True
     _description = "Search Engine Binding"
 
+    # Tech flag to identify model for SE bindings
+    _se_model = True
+    # Tech flag to identify model for SE bindings
+    # that do not require lang specific indexes.
+    # This flag does not trigger any automatic machinery.
+    # It provides a common key to provide implementers a unified way
+    # to check whether their specific binding models need or not lang spec index.
+    _se_index_lang_agnostic = False
```

Comment:
Why adding this flag when you can set lang_id = False  on records

Reply:
because you don't have any records yet. The only option would be to add a field to ir.model.field

Reply:
`_se_model` is used in the same fashion to retrieve such models